lib/ui.js

```diff
@@ -9,7 +9,28 @@ module.exports = options => {
 	const pkg = util.readPkg();
 	const oldVersion = pkg.version;
 
-	console.log(`\nPublish a new version of ${chalk.bold.magenta(pkg.name)} ${chalk.dim(`(${oldVersion})`)}\n`);
+	console.log(`\nPublish a new version of ${chalk.bold.magenta(pkg.name)} ${chalk.dim(`(current version: ${oldVersion})`)}\n`);
```

Comment:
@sindresorhus Should we revert these changes as it's unrelated to the issue. Should be discussed first at #116 imo.